vars/stageScanForSnyk.groovy

```diff
@@ -0,0 +1,46 @@
+def call(def context, def snykAuthenticationCode, def buildFile, def organisation) {
+  if (!snykAuthenticationCode) {
+    println("Skipping Snyk Scan due to missing authentication code (to enable pass as parameter a snyk service account authentication code to this stage)")
+    return
+  }
+  if (!organisation) {
+    organisation = context.projectId
+    println("organisation set to ${organisation}")
+  }
+
+  String message = "Snyk scan mode: build will " + (context.failOnSnykScanVulnerabilities ? "" : "not ") +
+  "fail if vulnerabilities are found (failOnSnykScanVulnerabilities=${context.failOnSnykScanVulnerabilities})!"
+  println(message)
+
+  if (buildFile == null) {
```

Comment:
Let's change to `if (!buildFile)` which also catches an empty string.

Reply:
done